code/src/components/NameQ.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const NameQ = ({ name, OnNameInputChange, nextQ }) => {
+  return (
+    <form onSubmit={nextQ}>
+      <label htmlFor="nameInput"><h2>Hi there! What is your name?</h2>
+        <input
+          id="name"
+          type="text"
+          name="name"
+          placeholder="Type name here..."
+          value={name}
+          onChange={OnNameInputChange}
+          autoComplete="off"
+          required />
```

Comment:
I totally forgot to add that it would be required to type in a name before going to next question.
Happy to see that you implement it!